Detectors/MUON/MID/Base/src/Constants.cxx

```diff
@@ -31,5 +33,18 @@ void Constants::assertDEId(int deId)
   }
 }
 
+//_____________________________________________________________________________
+std::string Constants::getDEName(int deId)
```

Comment:
I wonder whether the Constants class is the right location for all the getXXX functions. Why not make them free functions simply ? (they are not likely to clash with anything as they are within the o2::mid namespace already)

Reply:
Hi @aphecetche ,
you're right, I could well move the getter outside of the class (they're all static anyways).
However, this needs to be propagated in many places inside my code. I will therefore do a dedicated PR once this one is pushed.
Cheers,
Diego

Reply:
ok